pkg/k8s/types.go

```diff
@@ -70,6 +70,11 @@ const (
 	MeshRootCertificate InformerKey = "MeshRootCertificate"
 )
 
+const (
+	// kindSvcAccount is the ServiceAccount kind
+	kindSvcAccount = "ServiceAccount"
```

Comment:
Are there multiple serviceAccount kinds?

Reply:
No. I actually don't get the question.

Reply:
Usually, with a const grouping like you have here:
```go
const (
...
) 
```
each of the consts declared represent various options of a common type/concept. Looking again, I'm guessing this const is just here to deduplicate the string "ServiceAccount"?